reana_server/status.py

```diff
@@ -656,13 +675,17 @@ def get_workflow_health(self):
         running = wf_status_obj.get_workflows_by_status(RunStatus.running)
         queued = wf_status_obj.get_workflows_by_status(RunStatus.queued)
         pending = wf_status_obj.get_workflows_by_status(RunStatus.pending)
-        total = running + queued + pending
-        percentage = ClusterHealth.get_percentage(queued + pending, total)
+        total = REANA_MAX_CONCURRENT_BATCH_WORKFLOWS
+        used = running + pending
+        available = ClusterHealth.get_available(used, total)
+        percentage = ClusterHealth.get_percentage(used, total)
 
         return {
             "running": running,
             "queued": queued,
```

Comment:
Just thinking out loud, since `queued` workflows are not represented in the pie chart anymore, should we display them in a more prominent way?

Reply:
Good question. Here is how it looks like now:
![Screenshot from 2021-12-07 15-58-44](https://user-images.githubusercontent.com/4990025/145052832-38fa0be2-05d8-433e-b8f5-dab90dacaa3d.png)

Currently 10 queued workflows are not displayed in the chart and are not included in the available workflows and percentage calculation because of a bit different nature. So indeed it would be nice to display it in a more prominent way. I thought about using `italic` or `bold`, but it does not look great.. So I'm not sure, perhaps it's ok like this? Do you have any ideas? cc @tiborsimko

Reply:
The "queued" workflow information is actually rather imporant information for end users. Imagine we have 30 slots, and somebody (User A) submits 100 workflows.  Those 70 in-excess workflows can stay in the queue for a long time. They won't be running, they won't be pending, just queued.  Now another User B could estimate wow, the cluster is busy, I won't submit my workflows now.  (Which could be paradoxically not good, since our "balanced" workflow scheduling policy would give priority to User B!  But it's hard to express this on a simple one-plot-for-all graph.)

As a simple solution, in the above graph, I would simply print "10 queued" or bold and red colour.  But perhaps we should include them in the cirle plot.  I.e. 30 slots are available, but if circle plot has 100 things to print, everything above 30 will be painted red.  This will make the health status to be direcrly visible out-of-the-box.

Reply:
Queued workflows are now displayed in a more prominent way (bold and [`queued` color](https://github.com/reanahub/reana-ui/blob/3b25cc4d6b971956b8f62e3d0bdb44f6b9e15f9e/reana-ui/src/util.js#L28)) when `queued` > 0. Changes made in [the UI PR](https://github.com/reanahub/reana-ui/pull/209).